data/schemas/schemas-microsoft-com_office_powerpoint.json

```diff
@@ -0,0 +1,27 @@
+{
+  "TargetNamespace": "urn:schemas-microsoft-com:office:powerpoint",
+  "Types": [
+    {
+      "Name": "pvml:CT_Empty/pvml:iscomment",
```

Comment:
Just a question: why Name property is type/element like this. Should it be two separate properties?

Reply:
space saving :) Makes the file about 10% smaller

Reply:
What is "10%" in terms of absolute space (roughly)? The question is whether the 10% is worth the compromise. If you wanted to save space, you could also use shorter JSON property (key) names (e.g., `"tns"` instead of `"TargetNamespace"`.

Reply:
Space savings is in regards to line count

Also, this property is a "key" to which element it is - I believe (and my memory could be wrong here as I just got back from a vacation) that this combination leads to a unique lookup for the types.

Reply:
Ok. I just wasn't sure if this was needed by design. I prefer to have separate properties but it sounds like this was needed.